test/python/visualization/test_circuit_matplotlib_drawer.py

```diff
@@ -144,3 +144,29 @@ def test_long_name(self):
             'visualization/references/matplotlib_long_name_ref.png')
 
         self.assertImagesAreEqual(ref_filename, long_name_filename)
+
+    @unittest.skipIf(not visualization.HAS_MATPLOTLIB,
+                     'matplotlib not available.')
+    @unittest.skip('Test is unreliable on some platforms')
```

Comment:
Is there a reason to add the test if we're just skipping it everywhere?

Reply:
I think #2949 is removing the skip.